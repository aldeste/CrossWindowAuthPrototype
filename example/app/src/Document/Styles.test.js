const mockFunctionInjectGlobal = jest.fn();
jest.mock("styled-components", () => {
  return {
    injectGlobal: args => mockFunctionInjectGlobal(args)
  };
});

describe("Styles", () => {
  const InjectGlobalStyles = require("./Styles").default;
  it("Matches previous settings", () => {
    InjectGlobalStyles();
    expect(mockFunctionInjectGlobal.mock.calls[0]).toMatchSnapshot();
  });
  it("Runs without crashing", () => {
    expect(() => InjectGlobalStyles()).not.toThrowError();
  });
  it("Returns void", () => {
    expect(InjectGlobalStyles()).toBeUndefined();
  });
});
