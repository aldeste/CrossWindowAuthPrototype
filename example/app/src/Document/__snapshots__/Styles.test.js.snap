// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles Matches previous settings 1`] = `
"function InjectGlobalStyles() {
  _styledComponents.injectGlobal\`
  * {
    box-sizing: border-box;
    margin: 0;
    padding: 0;
  }

  *::after, *::before {
    box-sizing: inherit;
  }
\`;
}"
`;
